### PR TITLE
update axios version

### PR DESCRIPTION
**Pull request by narayana-plivo**

### CODE REVIEW

**Review by Vishnukumarpg — APPROVED** [review]

**Review by PratikKataria-plivo — COMMENTED** [review]
There are breaking changes in between 3 versions:
- https://github.com/axios/axios/releases/tag/v0.27.0
- https://github.com/axios/axios/releases/tag/v0.25.0
- https://github.com/axios/axios/releases/tag/v0.24.0

Are these validated?

File: types/rest/client.d.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ export function validateSignature(uri: string, nonce: string, signature: string,
  * You can also pass in additional parameters accepted by the node requests module.
  */
 export class Client {
-    constructor(authId?: string, authToken?: string, options?: string);
+    constructor(authId?: string, authToken?: string, options?: any);
```
Comment (kalyan-plivo) [inline types/rest/client.d.ts:10]: @narayana-plivo I believe the same line change we need to update in the **client-test.d.ts** file right?

Comment (narayana-plivo) [reply types/rest/client.d.ts:10]: @kalyan-plivo currently its passed with  constructor(authid: string, authToken: string, proxy: string); Not object.

**Comment by narayana-plivo** [comment]
> There are breaking changes in between 3 versions:
> 
> * https://github.com/axios/axios/releases/tag/v0.27.0
> * https://github.com/axios/axios/releases/tag/v0.25.0
> * https://github.com/axios/axios/releases/tag/v0.24.0
> 
> Are these validated?

i have verified most of the our api.

**Review by PratikKataria-plivo — APPROVED** [review]

**Comment by alexnault** [comment]
Hey Plivo Team! What's holding this PR from being merged?

**Pull request merged**